clients/web/src/app.js

```diff
@@ -1,43 +1,82 @@
 girder.App = girder.View.extend({
-    initialize: function () {
-        girder.fetchCurrentUser()
-            .done(_.bind(function (user) {
-                girder.eventStream = new girder.EventStream({
-                    timeout: girder.sseTimeout || null
-                });
+    /**
+     * @param {object} [settings]
+     * @param {bool} [settings.start=true] Run start after initialization
+     */
+    initialize: function (settings) {
+        this._started = false;
+        settings = settings || {};
+        if (settings.start === undefined || settings.start) {
+            this.start();
```

Comment:
Yes, that's the idea.  I wanted to decouple the construction of the object from starting the application.  Start is only called here to retain backward compatibility.  From what I've seen, it's a fairly common practice to have an `app.start` method ([`Marionette.Application`](http://marionettejs.com/docs/v2.4.5/marionette.application.html) for example).